camel-idea-plugin/src/main/java/com/github/cameltooling/idea/service/extension/camel/YamlCamelIdeaUtils.java

```diff
@@ -146,12 +147,12 @@ public PsiElement getPsiElementForCamelBeanMethod(PsiElement element) {
 
     @Override
     public List<PsiElement> findEndpointUsages(Module module, Predicate<String> uriCondition) {
-        return null;
+        return new ArrayList<>();
```

Comment:
You can likely use Collections.EMPTY_LIST() instead

Reply:
Ah, right, I was thinking about this, but for some reason didn't do it, let me make this change. This is all just temporary, until the full YAML support is done.